ccmpred/sampling/__init__.py

```diff
@@ -156,7 +156,7 @@ def sample_with_mutation_rate(tree, nseq, seq0, x, mutation_rate):
 
     return msa_sampled, neff
 
-def sample_to_neff_increasingly(tree, nseq, target_neff, ncol, x, gibbs_steps):
+def sample_to_neff_increasingly(tree, nseq, target_neff, ncol, x, gibbs_steps, root_seq = None):
```

Comment:
```suggestion
def sample_to_neff_increasingly(tree, nseq, target_neff, ncol, x, gibbs_steps, root_seq None):
```
Just a linting suggestion :)